scalanet/discovery/src/io/iohk/scalanet/discovery/ethereum/codecs/DefaultCodecs.scala

```diff
@@ -38,6 +39,21 @@ object DefaultCodecs {
       (sm: SortedMap[K, V]) => sm.toList
     )
 
+  class ByteIterableOrdering extends Ordering[Iterable[Byte]] {
+    import scala.math.Ordering._
+    final def compare(as: Iterable[Byte], bs: Iterable[Byte]) = {
+      val ai = as.iterator
+      val bi = bs.iterator
+      var result: Option[Int] = None
+      while (ai.hasNext && bi.hasNext && result.isEmpty) { 
+        val res = Byte.compare(ai.next(), bi.next())
+        if (res != 0) result = Some(res)
+      }
+      result.getOrElse( Boolean.compare(ai.hasNext, bi.hasNext) )
+    }
+  }
+
+  implicit val byteIterableOrdering = new ByteIterableOrdering
   implicit val byteVectorOrdering: Ordering[ByteVector] =
```

Comment:
Unfortunately I had to add the implementation because 2.12 and 2.13 orderings came from different packages. 
I will continue researching the possibility of ordering without triggering deprecation warnings.
I have also moved this monster to `util`ity package.